elastic/integrations
# Remove cgroups from system/process

## What does this PR do?

This PR removes the container dashboards and drops cgroup data from `system/process` in effort to save disk space. See https://github.com/elastic/integrations/issues/310 for more background.
## Checklist
 

## How to test this PR locally

- Pull down and build the integration
- Collect system/process data via agent
- Make sure no `cgroup` data is reported in `system.process`

## Related issues

- closes https://github.com/elastic/integrations/issues/310
